src/h2load.h

```diff
@@ -253,6 +255,15 @@ struct Worker {
   ev_timer timeout_watcher;
   // The next client ID this worker assigns
   uint32_t next_client_id;
+  // This variable tells us whether the client is in warmup phase or not or is over
+  // 0 - Not in warm-up phase, this is the initial state in timing experiment
+  // 1 - In warmup phase. This happens after the first connect. 
+  //   - All statistics are skipped/reversed in this phase
+  // 2 - Main duration phase, in timing experiment; Otherwise, the normal phase
+  // 3 - Main duration is over
+  int warmup;
```

Comment:
Yes, I also wanted a single phase but this is why I did both in the client and worker:
The worker keeps track of its phase and spawn all the clients. The client also needs to keep track of its phase because all the statistics are being measured inside their functions. Then once a client finishes, the worker's phase is updated. Then the worker informs all other clients to stop measurements. Until now, this is how I did it. 
We need to discuss it more for a better strategy.